.rdoc_options

```diff
@@ -0,0 +1,8 @@
+exclude:
+  - lib/rdoc/rd/block_parser.ry
+  - lib/rdoc/rd/inline_parser.ry
+  - lib/rdoc/markdown.kpeg
+  - lib/rdoc/markdown/literals.kpeg
+op_dir: _site # for GitHub Pages and should match the config of RDoc task in Rakefile
```

Comment:
`--op-dir` option is not saved by `--write-options`.
Is this expected?

Reply:
I don't think it is. But the bigger problem is that the presence of options (and some of their names) are different across Rake task, command line, and `.rdoc_options`. I think we should do a review and consolidation on them next year.